src/main/java/com/uci/inbound/incoming/CampaignController.java

```diff
@@ -52,9 +54,16 @@ public class Campaign {
     String topicFailure;
 
     @RequestMapping(value = "/start", method = RequestMethod.GET)
-    public ResponseEntity<String> startCampaign(@RequestParam("campaignId") String campaignId) throws JsonProcessingException, JAXBException {
+    public ResponseEntity<String> startCampaign(@RequestParam("campaignId") String campaignId, @RequestParam(value = "page", required = false) String page) throws JsonProcessingException, JAXBException {
+        Map<String, String> meta;
+        if(page != null && !page.isEmpty()){
+            meta = new HashMap<>();
+            meta.put("page", page);
+        } else {
+            meta = null;
+        }
```

Comment:
I think, else is redundant.

Reply:
Can we initialize the map unconditionally and just set the `page` value conditionally inside if? Then in the orchestrator service [here](https://github.com/pankajjangid05/orchestrator/blob/1839c52d1fb4f3b4c8af3b11054114569d3db782/src/main/java/com/uci/orchestrator/Consumer/ReactiveConsumer.java#L252) we do not need to check if page is null, we can just call `get("page")` and it'll pass null automatically if the `page` key is not set.